### DISCUSSION TITLE
Filtering and searching for the pull requests not working properly in Github

### DISCUSSION THREAD

**Discussion by samikhyaabi** [community/community#42291] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

We wanted to create a matrix on the basis of merged Pull requests which are approved before merging for a given date range.
We are facing following issues for the same:

**1. Approved and Merged PR search issue:** 
 So we applied following search query in pull requests search bar. But it is not reflecting the all the merged pull requests having "approved" tag. There are some merged PRs having tag "review required" is getting reflected in the results of search query for 
approved and merged PR list.

 Search query for the Merged PRs with "approved" tags
`is:pr is:closed merged:2022-08-25..2022-11-25 review:approved `
 
![image](https://user-images.githubusercontent.com/90616490/208608263-4728e7bd-0693-4d82-b099-779027e862f5.png)

**2.  Review Required and Merged PR search issue:**
  So we applied following search query in pull requests search bar. But it is not reflecting the all the merged pull requests having "review required" tag. There are some merged PRs having tag "Approved" is getting reflected in the results of search query for "review required" and merged PR list.

Search query for the Merged PRs with "review required" tags
`is:pr is:closed merged:2022-08-25..2022-11-25 review:required `

![image](https://user-images.githubusercontent.com/90616490/208608707-eafd7361-dacc-472b-b955-fd20f9469d2b.png)

We have referred following documentation for using search and filter tags.
https://docs.github.com/en/issues/tracking-your-work-with-issues/filtering-and-searching-issues-and-pull-requests

**Comment by Dubbeditt** [comment]


**Reply by coszen** [reply]
Can you please elaborate this ?
